CMakeLists.txt

```diff
@@ -2605,12 +2605,12 @@ find_package(LibUSB)
 # USB HID controller support
 option(HID "USB HID controller support" ON)
 if(HID)
-  find_package(hidapi)
-  # hidapi_VERSION is only available starting with 0.10.0
-  if(NOT hidapi_FOUND OR NOT hidapi_VERSION OR hidapi_VERSION VERSION_LESS 0.10.1)
```

Comment:
This change is unrelated to the fix and can be reverted.

Reply:
It is not unrelated. `find_package(hidapi 0.10.1)` creates the error message hinting the version is too old.